README.md

```diff
@@ -46,6 +46,9 @@ See the example below:
 For example value `-0.10` will cause immediate sell if the
 profit dips below -10% for a given trade. This parameter is optional.
 
+`opentradetimeout` is the number of minutes until an open trade will be cancelled. 
```

Comment:
The new documentation PR just merged conflicts with the README.md.
Could you please add these two lines in https://github.com/gcarq/freqtrade/blob/develop/docs/bot-usage.md?

Thanks

Reply:
Fixed!